setup.py

```diff
@@ -33,6 +33,7 @@
     "plecos",
     "cryptography==3.3.2",
     "scipy",
+    "enforce-typing"
```

Comment:
can you add a version to this? I know we weren't very consistent on this, but anytime is good to start being consistent. I think you can add the local version you're using, e.g. run `pip freeze` and grab it from there.